src/page/utkast/skjema-section/felt-header/felt-header.less

```diff
@@ -7,5 +7,10 @@
 		&__tittel {
 			margin-right: 1rem;
 		}
+
+		&__lenke {
+			margin-left: 20%;
```

Comment:
Jeg måtte søke litt rundt, men fant en ålreit måte å få den til å flytte seg til høyre på:
```css
    margin-left: auto;
    display: inline-flex;
    align-items: center;
    gap: 0.25rem;
```
De tre siste linjene gjør at ikonet midtstiller seg vertikalt, for nå stikker det av en eller annen litt grunn over teksten 🤔 Har slitt litt med de ikonene selv andre steder